core/server/worker/src/main/java/alluxio/worker/netty/DataServerUfsFileWriteHandler.java

```diff
@@ -47,16 +48,21 @@
     private final String mUfsPath;
     private final UnderFileSystem mUnderFileSystem;
     private final OutputStream mOutputStream;
+    private final Counter mCounter;
 
     FileWriteRequestInternal(Protocol.WriteRequest request) throws Exception {
       super(request.getId());
       Protocol.CreateUfsFileOptions createUfsFileOptions = request.getCreateUfsFileOptions();
       mUfsPath = createUfsFileOptions.getUfsPath();
-      mUnderFileSystem = mUfsManager.get(createUfsFileOptions.getMountId()).getUfs();
+      Ufs ufs = mUfsManager.get(createUfsFileOptions.getMountId());
+      mUnderFileSystem = ufs.getUfs();
       mOutputStream = mUnderFileSystem.create(mUfsPath,
           CreateOptions.defaults().setOwner(createUfsFileOptions.getOwner())
               .setGroup(createUfsFileOptions.getGroup())
               .setMode(new Mode((short) createUfsFileOptions.getMode())));
+      String ufsName = ufs.getUfsUri().toString().replace("/", "_");
+      String metricName = String.format("BytesWrittenUfs-Ufs:%s", ufsName);
```

Comment:
The logic to convert from a `Ufs` to the metric name should reside either in the `Ufs` or some utils method?